CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 This file documents all updates and releases to the ResearchNow Data Ingest pipeline.
 
+## [0.0.5] - Unreleased
+### Added
```

Comment:
I see that the ticket about adding edition identifiers to search results and work details is closed - was that done here or was it already available in the api?

Reply:
It's already available in the API, each `edition` object has an `id` attribute. I think it was included originally to help with React `key`s, but it is the right identifier to hit the endpoint